schemas/app-interface/database-access-1.yml

```diff
@@ -0,0 +1,39 @@
+---
+"$schema": /metaschema-1.json
+version: '1.0'
+type: object
+
+additionalProperties: false
+properties:
+  "$schema":
+    type: string
+    enum:
+    - /app-interface/database-access-1.yml
+  name:
+    type: string
+  username:
+    type: string
```

Comment:
type `string` can contain spaces, and I assume spaces and other characters aren't allowed in a username. please choose another type

Reply:
Good point! I addressed this